server/container_create.go

```diff
@@ -393,6 +393,11 @@ func (s *Server) getAppArmorProfileName(annotations map[string]string, ctrName s
 	}
 
 	if profile == apparmor.ProfileRuntimeDefault {
```

Comment:
Let me put it like this, since I don't think I'm explaining myself.

* `config.ApparmorProfile` tells ocid what is the name of the default profile to use when a container is requested to be started with `ProfileRuntimeDefault`.

* Currently, this code will load `ocid-default` **even if `config.ApparmorProfile != "ocid-default"**.

* In addition, it will load a profile that we generated, that the user probably doesn't want because it won't be used by anything.